app/javascript/shared/helpers/KeyboardHelpers.js

```diff
@@ -10,6 +10,10 @@ export const hasPressedShift = e => {
   return e.shiftKey;
 };
 
+export const hasPressedCommand = e => {
```

Comment:
Add spec

Reply:
@muhsin-k  Done https://github.com/chatwoot/chatwoot/pull/4401/commits/b7e285bc8cf0acdcd0de0b15826c5209db83fef7. Do I want to add specs for other helpers that have not been covered?